src/containers/BooksForm.js

```diff
@@ -1,14 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../actions/index';
 
 const BooksForm = () => {
+  const [book, setBook] = useState(
+    {
+      title: '',
+      category: 'Biography',
+    },
+  );
+
+  const handleChange = (e) => {
+    setBook({
+      category: e.target.value,
+    });
+  };
+
+  const handleInput = (e) => {
+    setBook({
+      title: e.target.value,
+    });
+  };
```

Comment:
You can have only one handleChange function: 

```
handleChange({ target: { name, value } }) {
  this.setState({
    [name]: value,
  })
}
```
But you have to update your form. Add name attribute to input tag and select tag, then in the option tag add the value attribute.